src/components/safe-apps/AddCustomSafeAppCard/AddCustomSafeAppCard.tsx

```diff
@@ -1,41 +1,39 @@
 import { useState } from 'react'
+import Card from '@mui/material/Card'
+import Box from '@mui/material/Box'
+import Button from '@mui/material/Button'
 import type { SafeAppData } from '@safe-global/safe-gateway-typescript-sdk'
-import { Button, Box } from '@mui/material'
-import { AddCustomAppModal } from '@/components/safe-apps/AddCustomAppModal'
-import { AppCardContainer } from './AppCard'
+
 import AddCustomAppIcon from '@/public/images/apps/add-custom-app.svg'
+import { AddCustomAppModal } from '@/components/safe-apps/AddCustomAppModal'
 
 type Props = { onSave: (data: SafeAppData) => void; safeAppList: SafeAppData[] }
 
-const AddCustomAppCard = ({ onSave, safeAppList }: Props) => {
-  const [addCustomAppModalOpen, setAddCustomAppModalOpen] = useState(false)
+const AddCustomSafeAppCard = ({ onSave, safeAppList }: Props) => {
+  const [addCustomAppModalOpen, setAddCustomAppModalOpen] = useState<boolean>(false)
 
   return (
     <>
-      <AppCardContainer>
-        <Box
-          sx={{
-            height: '100%',
-            display: 'flex',
-            flexDirection: 'column',
-            alignItems: 'center',
-            justifyContent: 'center',
-          }}
-        >
-          <AddCustomAppIcon />
+      <Card>
+        <Box padding="48px 12px" display="flex" flexDirection="column" alignItems="center">
+          {/* Add Custom Safe App Icon */}
+          <AddCustomAppIcon alt="Add Custom Safe App card" />
+
+          {/*  Add Custom Safe App Button */}
           <Button
             variant="contained"
             size="small"
+            onClick={() => setAddCustomAppModalOpen(true)}
             sx={{
-              mt: 1,
+              mt: '24px',
```

Comment:
why hardcode the value instead of using the scaling factor?

Reply:
done!  `mt: '24px'`  ->  `mt: 3`